azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/hangup/LeaveConfirmView.kt

```diff
@@ -104,15 +74,32 @@ internal class LeaveConfirmView(
 
     private fun initializeLeaveConfirmMenuDrawer() {
         leaveConfirmMenuDrawer = DrawerDialog(context, DrawerDialog.BehaviorType.BOTTOM)
+        leaveConfirmMenuDrawer.setContentView(this)
         leaveConfirmMenuDrawer.setOnDismissListener {
             viewModel.cancel()
         }
-        leaveConfirmMenuDrawer.setContentView(this)
+    }
+
+    private fun cancelLeaveConfirm() {
+        leaveConfirmMenuDrawer.dismiss()
     }
 
     private val bottomCellItems: List<BottomCellItem>
         get() {
             val bottomCellItems = mutableListOf(
+                // Leave title
+                BottomCellItem(
+                    null,
+                    context.getString(R.string.azure_communication_ui_calling_view_leave_call),
+                    context.getString(R.string.azure_communication_ui_calling_view_leave_confirm_menu),
```

Comment:
can we rename this string to `azure_communication_ui_calling_view_leave_confirm_content_desciption` for better clarity?

Reply:
or "azure_communication_ui_calling_view_leave_confirm_title_description"

Reply:
Included in 0beaa7d